src/singleton.js

```diff
@@ -72,5 +74,14 @@ export default class Singleton extends Tracer {
 
         // Carrier objects to be exposed at the package level
         this.BinaryCarrier = BinaryCarrier;
+
+        // XXX: what's the best way to export (and document) these two global
+        // helper functions?
+        this.childOf = function(spanContext) {
+            return new Reference(Constants[REFERENCE_CHILD_OF], spanContext);
+        };
+        this.followsFrom = function(spanContext) {
+            return new Reference(Constants[REFERENCE_FOLLOWS_FROM], spanContext);
```

Comment:
I don't think this is right. I assume you mean either `Constants.REFERENCE_CHILD_OF` or `Constants['REFERENCE_CHILD_OF']`.